code/modules/clothing/neck/_neck.dm

```diff
@@ -193,9 +193,11 @@
 	var/tagname = null
 
 /obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
-	if(ishuman(M))
-		return FALSE
-
+	if(istype(M, /mob/living/carbon/human/species/felinid)) // yogs start - lets catpeople wear collars
```

Comment:
you do realize we don't use this mob for purrbation, meaning this only affects people you've spawned in with this mobtype, right?